src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -85,10 +123,17 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 			tooltip = tooltip + 'Login: ' + connectionProfile.userName + '\r\n';
 		}
 
+		if (pid) {
+			text += ' (' + pid + ')';
+			let processIDName = connectionProfile.serverCapabilities.processIDName;
```

Comment:
nit: I'd combine the default value into this
```suggestion
			const processIDName = connectionProfile.serverCapabilities.processIDName || 'PID';
```